packages/venia-ui/lib/components/Navigation/navigation.css

```diff
@@ -74,7 +74,9 @@
 }
 
 .switchers {
-    display: block;
+    display: grid;
+    grid-template-columns: auto auto;
+    justify-content: space-between;
```

Comment:
If all the values are `auto` then you can generally omit `grid-template-columns` in favor of `grid-auto-flow: column`. The result will better accommodate new grid items.

Also, consider using a fixed `gap` instead of `space-between`.